CONTRIBUTING.md

```diff
@@ -50,9 +50,10 @@ Every Issue and PR should be tagged with one or more of the following labels to
 
 <a name="prs"></a>
 ### Pull Requests
-- PRs must only be created when there is an issue present and a decision to create a PR is made. A PR must reference the issue it is resolving in the description field.
-  - using GitHub predefined keyword "**Fixes Issue#**" will automatically close the issue which the PR is resolving when it is merged.
-- A PR for an issues must only be created by the contributor assigned to the issue by the respective WSL.
+- PRs dealing with minor editorial changes can be raised without a corresponding Issue, but must be labelled as "Minor". Otherwise, PRs must only be created when there is an issue present and a decision to create a PR is made. 
```

Comment:
Can we clarify "minor editorial"? Is it spelling or grammatical changes or does it also include more substantive policy/meaning changes? For example, this PR proposes a change to the process but is classified as "minor".

Reply:
definition of "minor" is indeed

Reply:
Perhaps we should have another PR label like "Agreed" which would qualify for being raised without an issue?  This is the case with the example above: I would agree it is not "Minor" (it changes the process, as @pgoyal01 wrote) but it was agreed in TSC so it can fast processed.

Reply:
The minor label already had a definition,:
> Pull request suggesting minor/editorial changes

I don't set the problem with this PR being labelled as minor as it is a small change,  but as it is not editorial it had a corresponding issue. In other words I take the `/` in the definition to be an `or`. It doesn't mean, to me,  that all PRs labelled as minor are only editorial and without an issue.

Reply:
@ASawwaf @walterkozlowski @pgoyal01 WSLs are expected to approve minor PRs as any other PR (the only difference is that minor PRs can be created without an issue), as @tomkivlin said, they can still have issue but not required.

In any case, if WSL feel the PR was not eligible to be considered minor, they can push back and force an issue to be created for it. There will be no accurate definition of minor and I am not very keen to try and define it as this will be a long discussion I feel. let us follow the common sense here and consider PRs on a case by case basis.